src/index.ts

```diff
@@ -1,24 +1,22 @@
 import express, { Application, Request, Response } from "express";
 import bodyParser from "body-parser";
 import mongoose from "mongoose";
-import cors from "cors";
-import { todoRouter } from "./routes/todo";
-import { userRouter } from "./routes/userRouter";
+// import cors from "cors";
+import { config } from "dotenv";
+import { todoRouter } from "@/routes/todo";
+import { userRouter } from "@/routes/userRouter";
```

Comment:
if you set "baseUrl" : "src" then you can import todoRouter from "routers/todo"
No need to add "paths" in tsconfig file

Reply:
same for other files